spec/features/evidence/evidence_check_flow_spec.rb

```diff
@@ -101,4 +101,30 @@
       it { expect(page).to have_xpath('//div[contains(@class,"callout-full")]/h3[@class="bold"]', text: '✓ The applicant doesn’t have to pay the fee') }
     end
   end
+
+  context 'when on "Evidence confirmation" page' do
+    before { visit evidence_confirmation_path(id: evidence.id) }
+
+    it { expect(page).to have_content 'Processing complete' }
+
+    context 'when the remission is' do
+      context 'full' do
+        let(:outcome) { 'full' }
+
+        it { expect(page).not_to have_content /(not\ correct\|part-fee)/ }
```

Comment:
Capybara has a special matcher for testing that something is not present on the page: `have_no_content()`, because using `not_to` doesn't take into account Javascript wait, etc.

But I think that this test may not be relevant anyway, because testing that a particular text is not on the page doesn't proof that the page is correctly rendered. Hope that makes sense.